lib/generators/templates/rpush_9_1_0_add_hms.rb

```diff
@@ -0,0 +1,22 @@
+# migration 3_1_1_a is mapped to version 3.1.1.1
+class Rpush910AddHms < ActiveRecord::Migration[6.0]
+  def self.up
+    add_column :rpush_apps, :hms_app_id, :string
+    add_column :rpush_apps, :hms_key_id, :string
+    add_column :rpush_apps, :hms_sub_acc_id, :string
+    add_column :rpush_apps, :hms_key, :string
+    add_column :rpush_notifications, :title, :string
```

Comment:
Could you help me understand why this is required for HMS but not for other systems?